aesara/configparser.py

```diff
@@ -94,7 +95,7 @@ def __init__(self, flags_dict: dict, aesara_cfg, aesara_raw_cfg):
         self._flags_dict = flags_dict
         self._aesara_cfg = aesara_cfg
         self._aesara_raw_cfg = aesara_raw_cfg
-        self._config_var_dict = {}
+        self._config_var_dict: typing.Dict = {}
```

Comment:
Looks like some hints use a different import pattern

Reply:
Yes, this file already had an `import typing` call and I didn't want to change everything using the `from typing import Dict` pattern, but it would make sense to uniformize this at a certain point.